logzio/sender.py

```diff
@@ -153,12 +158,19 @@ def _flush_queue(self):
                     'backing up to local file system', number_of_retries)
                 backup_logs(logs_list, self.logger)
 
+            del logs_list
+
     def _get_messages_up_to_max_allowed_size(self):
         logs_list = []
         current_size = 0
         while not self.queue.empty():
             current_log = self.queue.get()
-            current_size += sys.getsizeof(current_log)
+            try:
+                current_size += sys.getsizeof(current_log)
+            except TypeError:
+                # pypy do not support sys.getsizeof
+                current_size += len(current_log) * 4
+
```

Comment:
@roiravhon what do you think?
I think the upper representation is 4 so in the worst case we will send a smaller bulks

Reply:
@idohalevi we can go all philosophical on that manner, but I think this should work.
Can we just verify with large data set that the bulks are still in the right size?